common/src/main/scala/explore/components/Tile.scala

```diff
@@ -19,18 +19,19 @@ import react.common.style.*
 import react.primereact.Button
 
 case class Tile(
-  id:                Tile.TileId,
-  title:             String,
-  back:              Option[VdomNode] = None,
-  control:           TileSizeState => Option[VdomNode] = _ => None,
-  canMinimize:       Boolean = false,
-  canMaximize:       Boolean = false,
-  state:             TileSizeState = TileSizeState.Normal,
-  sizeStateCallback: TileSizeState => Callback = _ => Callback.empty,
-  controllerClass:   Option[Css] = None, // applied to wrapping div when in a TileController.
-  bodyClass:         Option[Css] = None, // applied to tile body
-  tileClass:         Option[Css] = None, // applied to the tile
-  tileTitleClass:    Option[Css] = None  // applied to the title
+  id:                 Tile.TileId,
+  title:              String,
+  back:               Option[VdomNode] = None,
+  control:            TileSizeState => Option[VdomNode] = _ => None,
+  canMinimize:        Boolean = false,
+  canMaximize:        Boolean = false,
+  state:              TileSizeState = TileSizeState.Normal,
+  sizeStateCallback:  TileSizeState => Callback = _ => Callback.empty,
+  controllerClass:    Option[Css] = None, // applied to wrapping div when in a TileController.
+  bodyClass:          Option[Css] = None, // applied to tile body
+  tileClass:          Option[Css] = None, // applied to the tile
+  tileTitleClass:     Option[Css] = None, // applied to the title
+  renderInTitleClass: Option[Css] = None  // applied to the portal in the title
```

Comment:
This comment goes beyond the scope of this PR, but I wonder if we need `Option[Css]` here when we have `Css.Empty`.

Reply:
That's a good suggestion, I'll implement it